docs/extend/extend-apiml/api-mediation-routing.md

```diff
@@ -0,0 +1,94 @@
+# API Mediation Layer routing
+
+As an application developer, route your service to ...
+There are two ways to route to the API Mediation Layer:
+
+* API ML Basic Routing (Service ID and version-based)
+* Basic Routing (only service ID-based)
+
+## Terminology
+
+* **Service**
+
+  A service provides one or more APIs. A service is identified by its service ID (sometimes the term service name is used in the same     meaning). 
```

Comment:
Please remove unnecessary spces

Reply:
It is actually intentional as it is part of the Service bullet point